extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -656,6 +656,16 @@ export class AmpSlideScroll extends BaseSlides {
         this.getScrollLeftForIndex_(newIndex);
     this.triggerAnalyticsEvent_(newIndex);
     this.slideIndex_ = newIndex;
+    // If we have a specified number of autoplay loops and
+    // we have reached the last slide in the set
+    // carry out removing autoplay logic.
+    // This only works because the initial Slide is always the first slide.
+    if (this.autoplayLoops_ && this.slideIndex_ === this.noOfSlides_ - 1) {
```

Comment:
wha if `autoplayLoops_` is set to `0`? this logic would treat it as infinite loop. We probably want to either disallow it  or treat `autoplay=0` same as `no autoplay`

Reply:
Done. If the value of `autoplayLoops_` is set to `0` we never call `setupAutoplay_` or `autoplay_`